plugin/src/main/resources/io/jenkins/plugins/coverage/model/CoverageViewModel/index.jelly

```diff
@@ -3,7 +3,7 @@
 
   <st:header name="Content-Type" value="text/html;charset=UTF-8"/>
 
-  <bs:page it="${it}" class="fluid-container d-flex flex-column">
+  <bs:page it="${it}" class="fluid-container d-flex flex-column h-100">
```

Comment:
This breaks the visualization on smaller displays.

Reply:
Can we remove these unrelated changes? Then we can merge the color PR and create a new release.